README.md

```diff
@@ -10,46 +10,120 @@ It's important to note that the code in this repo may differ from the code writt
 ## Lab 00
 Running Fusion locally.
 
-1. Clone this repo
+1. Checkout your own branch
 ```
-git clone git@github.com:wapopartners/Fusion-Training-User-Stories.git
+git checkout USERNAME-fusion-training
 ```
 
-2. Change directory into Fusion-Training-User-Stories
+2. Install all packages/dependencies
 ```
-cd Fusion-Training-User-Stories
+npm install
 ```
 
-3. Checkout the `lab-00` branch
+3. To ensure you are using the latest fusion-cli, run:
 ```
-git checkout lab-00
+npm install @arc-fusion/cli@canary
```

Comment:
We want to keep them on the latest stable one so we dont have to have them install @canary

Reply:
So just `npm install @arc-fusion` ? Or just `npm install`?

Reply:
It might not be needed at all as npx... init might take care of this step - so walk through the new master instructions and see if this is needed